.environment-scripts/gaea-c5/activate_conda_environment.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+CONDA_ENV=$1
+
+module load python/3.9
+eval "$(conda shell.bash hook)"
```

Comment:
This actually works on the C5 partition because the `python/3.9` module does not modify the `LD_LIBRARY_PATH`.